tests/Smoke/InstallViaComposerTest.php

```diff
@@ -135,15 +135,15 @@ public function testInstallationViaArtifactIsPossible()
         $stepsToInitializeArtifact = [
             // Clone current version of project to new location, as we gonna modify it.
             // Warning! Only already committed changes will be cloned!
-            "git clone . {$tmpArtifactPath}",
+            "git clone --depth=1 . {$tmpArtifactPath}",
         ];
         $stepsToPrepareArtifact = [
             // Configure git user for new repo to not use global git user.
             // We need this, as global git user may not be set!
             'git config user.name test && git config user.email test',
             // Adjust cloned project to expose version in `composer.json`.
             // Without that, it would not be possible to use it as Composer Artifact.
-            "composer config version {$fakeVersion} && git add . && git commit -m 'provide version'",
+            "composer config version {$fakeVersion} && git add . && git commit --no-gpg-sign -m 'provide version'",
```

Comment:
isn't no-gpg-sign default , @sanmai ?

Reply:
Yes and no. You can configure Git to sign all commits everywhere, by default. And then this test will fail.